app/controllers/api/households_controller.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+
+module Api
+  # Exposes the Household data
+  class HouseholdsController < Api::BaseController
+    before_action :set_household, only: %i[show update delete]
+    # GET /households
+    # def index
+    #   # Should get household by current user_id/member_id
+    #   @household = Household.joins(:members)
+    #   .where(members: { id: current_user.member.id })
+    #   render json: @household
+    # end
+
+    # GET /households/1
+    def show
+      if @household
+        render json: @household
+      else
+        render json: {}, status: :not_found
+      end
+    end
+
+    # POST /households
+    def create
+      @household = Household.new(household_params)
+      set_added_by
+      if @household.save
+        render json: @household
+      else
+        render json: @household.errors, status: :unprocessable_entity
+      end
+    end
+
+    # PUT /households/1
+    def update
+      if @household.update(household_params)
```

Comment:
Do we need to handle the `last_updated_by`, etc fields here?

Reply:
@nvengal updated to call `set_added_by`, that should be it right?

Reply:
The `set_added_by` method updates the `added_by` and `updated_by` columns. That worked fine for the create method. But for the update method we only want to change the `updated_by` columns. So, I think you need to split `set_added_by` into two methods so that the `updated_by` columns can be changed independently.

Long term we should probably use a gem like https://github.com/paper-trail-gem/paper_trail for audit logging, but this is fine for now.

Reply:
Duh, total brainfart. Okay should be good now I believe